.tekton/pipeline-service-ckcp-testing.yaml

```
@@ -234,8 +240,8 @@ spec:
                 echo "Copy kubeconfig file to podman container"
                 kubectl cp "$(workspaces.kubeconfig-dir.path)/kubeconfig" default/podman:/kubeconfig
 
-                echo "Start to set up ckcp"
-                kubectl -n default exec pod/podman -- sh -c "export KUBECONFIG=/kubeconfig && /source/ckcp_setup.sh"
+                echo "Start to test ckcp"
+                kubectl -n default exec pod/podman -- sh -c "export KUBECONFIG=/kubeconfig && /source/ckcp_setup.sh "
```

Missed this whitespace.
```suggestion
                kubectl -n default exec pod/podman -- sh -c "export KUBECONFIG=/kubeconfig && /source/ckcp_setup.sh"
```